### PR TITLE
Sane numpy matrix

### PR DESCRIPTION
**Pull request by ocefpaf**
@jjhelmus I'm stealing your idea here :wink: 

I rebuilt everything from our internal system and it worked wonders! No reason not to use it in `conda-forge` too.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,14 +26,18 @@ requirements:
...
     - vc 9              # [win and py27]
     - vc 14             # [win and py>=35]
     - boost-cpp {{ version }}
 
   run:
     - python
-    - numpy x.x
+    - numpy 1.7.*  # [py27]
+    - numpy 1.9.*  # [py35]
+    - numpy 1.11.*  # [py36]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Aren't these suppose to be lower bounded not pinned to a major minor version? 😕

xref: https://github.com/conda-forge/pandas-feedstock/pull/32/files#diff-e178b687b10a71a3348107ae3154e44cR27

Comment (ocefpaf) [inline recipe/meta.yaml]: Only when the PR is ready. Not when I want to ensure that the same `numpy` is used to check the linkages and the `mkl` vs `openblas` situation.

Comment (jakirkham) [inline recipe/meta.yaml]: I don't think Boost is using a BLAS, but am all for being cautious. Thanks for checking. :)

File: recipe/meta.yaml [hunk 2]
```diff
@@ -26,14 +26,18 @@ requirements:
...
     - vc 9              # [win and py27]
     - vc 14             # [win and py>=35]
     - boost-cpp {{ version }}
 
   run:
     - python
-    - numpy x.x
+    - numpy >=1.7  # [py27]
+    - numpy >=1.9  # [py35]
+    - numpy >=1.11  # [py36]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: @jjhelmus I got `numpy 1.7.*` working with `openblas` for `py27` but not `py35`. I wonder if we should try `numpy 1.9` everywhere to simplify this. What do you think?

**Comment by isuruf** [comment]
@ocefpaf, can you open a PR for the docs repo?

**Comment by ocefpaf** [comment]
> @ocefpaf, can you open a PR for the docs repo?

Should've done that first :wink: 

Working on it now. I'll ping you for a review soon.

**Pull request merged**